client/app/components/editMoment.js

```diff
@@ -26,17 +25,18 @@ class EditMoment extends Component{
         <View style={styles.row}>
           <TouchableHighlight key={asset} onPress={() => {
               NativeModules.ReadImageData.readImage(image.uri, (image) => {
```

Comment:
Should pass parameters into readImage, might want to rename readImage because that's not what it does anymore